spezi-module/onboarding/src/main/java/edu/stanford/spezikt/spezi_module/onboarding/invitation/FirebaseInvitationAuthManager.kt

```diff
@@ -0,0 +1,60 @@
+package edu.stanford.spezikt.spezi_module.onboarding.invitation
+
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.functions.FirebaseFunctions
+import edu.stanford.spezi.logging.speziLogger
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.suspendCancellableCoroutine
+import javax.inject.Inject
+import kotlin.coroutines.resumeWithException
+
+
+class FirebaseInvitationAuthManager @Inject constructor() : InvitationAuthManager {
```

Comment:
I would name this `InvitationAuthManagerImpl` and mark internal

Reply:
personally, i like the current naming better because it tells the developer directly that it is a firebase specific implementation. also, since it is an interface and it is to be expected that other implementations will be added later, it makes identification easier. is there anything specifically against the current naming, @eldcn ?